apps/nsq_to_http/nsq_to_http.go

```diff
@@ -180,6 +183,14 @@ func main() {
 		return
 	}
 
+	if len(customHeaders) > 0 {
+		validCustomHeaders, err := parseCustomHeaders(customHeaders)
+		if err != nil {
+			log.Fatal("--header value format should be 'key=value'")
+		}
+		log.Printf("Specified custom headers: %v", validCustomHeaders)
```

Comment:
Can we drop this log line?

Reply:
Yes it does seem redundant. But technically I need help here. For some reason I keep getting an error that I have an unused variable called `validCustomHeaders`
I could use a lil help here

Reply:
Ahhh, an annoying quirk of Go known as "variable shadowing". What you want on line 187 is this:

```go
var err error
validCustomHeaders, err = parseCustomHeaders(customHeaders)
...
```

(note the lack of `:=`)

Reply:
I know Go does not allow you to have unused variables but "variable shadowing" I am hearing for the first time. I have fixed it.